### PR TITLE
Add code action jump from dependency in editor to tree

### PR DESCRIPTION
**Pull request by attiasas**
- [x] All [tests](https://github.com/jfrog/jfrog-vscode-extension#building-and-testing-the-sources) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

### CODE REVIEW

File: src/main/treeDataProviders/issuesTree/descriptorTree/descriptorTreeNode.ts [hunk 1]
```diff
@@ -49,11 +49,11 @@ export class DescriptorTreeNode extends FileTreeNode {
 
     /**
      * Search for registered dependency with issue in this descriptor base on a given artifactId.
-     * @param artifactId - the id (type,name,version) of the dependency to search
+     * @param artifactId - the artifactId (type,name,version)/ componentId (name,version) of the dependency to search
```
Comment (yahavi) [inline src/main/treeDataProviders/issuesTree/descriptorTree/descriptorTreeNode.ts:52]:
```suggestion
     * @param artifactId - the artifactId (type,name,version) or the componentId (name,version) of the dependency to search
```

File: src/main/treeDataProviders/utils/descriptorUtils.ts [hunk 2]
```diff
@@ -20,6 +20,11 @@ import { FocusType } from '../../constants/contextKeys';
 import { DescriptorIssuesData } from '../../types/issuesData';
 
 export class DescriptorUtils {
+    public static readonly DESCRIPTOR_SELECTOR: vscode.DocumentSelector = {
+        scheme: 'file',
+        pattern: '**/{go.mod,package.json,pom.xml,*requirements*.txt,*.sln,yarn.lock}'
```
Comment (yahavi) [inline src/main/treeDataProviders/utils/descriptorUtils.ts:25]: Why is the `*sln`? NuGet is not supported in getDependencyPosition

**Review by yahavi — CHANGES_REQUESTED** [review]

File: src/main/diagnostics/descriptorActionProvider.ts [hunk 3]
```diff
@@ -31,7 +31,60 @@ interface DirectDependencyIssue {
...
+                        }
+                    }
+                }
+            }
+            let commands: vscode.Command[] = [];
+            // Add jump to tree commands for each infected dependency
+            for (const infectedDependency of infectedDependencies) {
+                commands.push({
+                    command: 'jfrog.issues.select.node',
+                    title: 'Show infected dependency ' + infectedDependency.componentId + ' in issues tree',
```
Comment (yahavi) [inline src/main/diagnostics/descriptorActionProvider.ts:78]: Wrap the infected dependency id with quotes
```suggestion
                    title: "Show infected dependency '" + infectedDependency.componentId + "' in issues tree",
```

File: src/main/treeDataProviders/utils/descriptorUtils.ts (hunk 2 above)
Comment (attiasas) [reply src/main/treeDataProviders/utils/descriptorUtils.ts]: removed

**Pull request merged**